Kunena/Kunena-Forum
# Change: rename user in messages should also rename users in topics

Pull Request for Issue # . 
 
#### Summary of Changes
currently when syncing usernames in messages, it does just that: sync the username in messages. but the username is also set in topics and there it is not renamed.

This could (and if it can, it will) be a gdpr issue. When a user requests his account to be deleted. The joomla user is renamed.
after approving this, the administrator should sync the usernames in messages to set the new username, but the old username still shows on topics (first post / last post).

This PR changes that: after this PR also the usernames in topics are changed
 
#### Testing Instructions

update usernames and check if they are all replaced in messages, but also in topics.
